src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -275,9 +275,9 @@ public static Category parseCategory(String category) throws ParseException {
 
     /**
      * Parses a {@code String aliasName} and {@code String command} into a {@code Alias}.
-     * @param aliasName
-     * @param command
-     * @throws ParseException
+     * @param aliasName name of the alias
+     * @param command content of the command
```

Comment:
Please standardize the javadocs format in future to follow the style guide. Will approve first because it's non-breaking and we should merge soon.